### DISCUSSION TITLE
Trying to load template on SLEAP but it is not working

### DISCUSSION THREAD

**Discussion by CherieSun052** [talmolab/sleap#1402] [Help!]
Hi! I am trying to load the template and start the label but after I click load there is no response. What should I do to fix it?

**Comment by roomrys** [comment]
Hi @CherieSun052,

Do you mind providing some information on which template are you trying to load and how you are trying to load it? Also, are there any messages in your terminal?

Thanks,
Liezl

**Reply by CherieSun052** [reply]
yes! I am trying to load flies13 and I just open a video and click load. I think it shows: attributeError: "mainwindow" object has no attribute "skeletontemplates"

**Comment by roomrys** [comment]
Hi @CherieSun052,

This is a bug. You will need to create your own skeleton until it is resolved. I've writtten a pull request for this (see #1404).

Thanks,
Liezl

**Reply by roomrys** [reply]
Alternatively, you can download the skeleton template json that you want to load from [here](https://github.com/talmolab/sleap/tree/develop/sleap/skeletons) and select `Load From File...` at the bottom of the Skeleton dock.

**Reply by CherieSun052** [reply]
Hi, is there any update on the pull request? I am also having the same problem (this is clee758 by the way). Also, when I load the "flies32" json file, it keeps loading each part but many of them (for example more than one left midleg). Is there an issue with the file or is it probably something wrong with what I am doing?

**Reply by roomrys** [reply]
Hi @CherieSun052,

The PR has been merged and the fix will be available in the next release (v1.3.2). If you have installed SLEAP via [conda from source](https://sleap.ai/installation.html#conda-from-source), you can get the fixes prior to the release by navigating to the folder that contains the sleap source code and performing a `git pull`.

Thanks,
Liezl
